huaweicloud/resource_huaweicloud_compute_volume_attach.go

```diff
@@ -16,11 +17,11 @@ import (
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
 )
 
-func ResourceComputeVolumeAttachV2() *schema.Resource {
+func ResourceComputeVolumeAttach() *schema.Resource {
```

Comment:
If we changed the function name, joined providers should also be modified accordingly.